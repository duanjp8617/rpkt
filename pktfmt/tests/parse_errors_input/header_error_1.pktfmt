packet FakeProto {
    header = [
        field_name1 = Field {
            bit = 8, 
        },
        field_name1 = Field {
            bit = 8, 
        }
    ]
}

