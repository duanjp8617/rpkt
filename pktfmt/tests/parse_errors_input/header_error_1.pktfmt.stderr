ast error
at /home/djp/pkt-rs/pktfmt/tests/parse_errors_input/header_error_1.pktfmt 6:9-19
6 |         field_name1 = Field {
            ^^^^^^^^^^^
header error 1:
duplicated header field name field_name1
